### ISSUE TITLE
Need documentation on Internet access requirements for CFT to work

### ISSUE THREAD

**Issue by C0missar** [F5Networks/f5-aws-cloudformation#96]
## Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
Describe the problem you're having or the enhancement you'd like to request.
```
Readme.md (f5-existing-stack-across-az-cluster-byol-3nic-bigip.template) now cites a requirement for Internet access on both External and Management interfaces.

Please describe what each will be used for and what routes must be in place, particularly if deploying with ProvisionPublicIP=No, as building a pair now fails with ProvisionPublic IP=No.

It states that the external interface is used for "Onboarding", but there are no TMM routes built on the external interface, so it's not clear how this would work or what onboarding activities would take place there.

It also states that the management interface is used for downloading the BIG-IP cloud library.  What is that, do I need it, and why isn't it already on the AMI?

Which interface is used for licensing?  

What routes must be in place on each interface in order for these to work, and how are they established prior to CFT completion?
```
## Template
For bugs, enter the template with which you are experiencing issues below.

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: <Fill in level: 1 through 5>

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by shyawnkarim** [comment]
Thank you for providing this feedback around around our documentation. I've created internal issue ESECLDTPLT-1950 to enhance our documentation around these topics.

**Comment by shyawnkarim** [comment]
Closing. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).

Our documentation has been completely rebuilt for V2.
